apis/frontend/search-js/README.md

```diff
@@ -33,10 +33,85 @@ const response = await search({
     }
 }, {
     track: 'serp',
+    hitDecorators: [
+        priceDecorator(),
+        thumbnailDecorator({ size: "9" })
+    ],
+    maxRetries: 3,
+    retryInterval: 2000,
+    usePersistentCache: true,
+    useMemoryCache: false
+})
+
+```
+
+## Search Options
+
+The search function accepts all standard Nosto JS API options plus the following search-js specific options:
+
+### `hitDecorators`
+**Type:** `HitDecorator[]`  
+**Default:** `undefined`
+
+Hit decorators to apply to the search results. These allow you to transform search results with additional functionality like currency formatting and thumbnail generation.
+
```

Comment:
provide this as an example of a custom decorator that adds handle field to the product

```
import { SearchProduct } from "@nosto/nosto-js/client"

type DecoratedProduct = SearchProduct & { handle?: string }

export function handleDecorator(product: SearchProduct): DecoratedProduct {
  if (product.url) {
    const pathname = new URL(product.url).pathname
    return {
      ...product,
      handle: pathname.split("/").pop() || undefined
    }
  }
  return product
}
```